testrunner-running/src/main/scala/edu/illinois/cs/statecapture/agent/MainAgent.java

```diff
@@ -0,0 +1,42 @@
+package edu.illinois.cs.statecapture.agent;
+import java.lang.instrument.Instrumentation;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+   
+public class MainAgent {  
+    private static Instrumentation inst;
+    public static String xmlFold;
+    public static String subxmlFold;
+    public static String rootFold;
+    public static String pkgFile;
+    public static String diffFold;
+    public static String slug;
+    public static String targetTestName;
+    public static String outputPath;
+    public static String diffFieldFold;
+    public static String fieldFold;
+    public static String subdiffFold;
+    public static String tmpfile;
+    public static String reflectionFold;
+
+    public static Instrumentation getInstrumentation() { return inst; }
+   
+    public static void premain(String agentArgs, Instrumentation inst) {
+        MainAgent.inst = inst;
+        String[] args = agentArgs.split(",");
+        xmlFold = args[0];
```

Comment:
These options were useful for our experiments, but to keep it clean let's see if we can convert these to use some Configuration logic instead.